Bottom: f61b9bc702607ae2aca1f04aad36913ef80807a7
Top:    48d7fc6052cd80faecb17f8942171f834e9de0d8
Author: scott snyder <sss@karma>
Date:   2020-02-12 11:42:23 -0500

Fix compilation warning.


---

diff --git a/src/PandoraPFACalibrator.cc b/src/PandoraPFACalibrator.cc
index 7ea4373..444bb9f 100644
--- a/src/PandoraPFACalibrator.cc
+++ b/src/PandoraPFACalibrator.cc
@@ -548,7 +548,7 @@ void PandoraPFACalibrator::ReadPfoCollections(LCEvent *pLCEvent, const LCStrVec
 
 //------------------------------------------------------------------------------------------------------------------------------------------
 
-void PandoraPFACalibrator::check(LCEvent *pLCEvent)
+void PandoraPFACalibrator::check(LCEvent */*pLCEvent*/)
 {
 }
